http/client.go

```diff
@@ -129,36 +130,46 @@ func (dd *debugDoer) Fprint(w io.Writer) {
 }
 
 // Error builds an error message.
-func (c *clientError) Error() string {
-	return fmt.Sprintf("[%s %s]: %s", c.service, c.method, c.message)
+func (c *ClientError) Error() string {
+	return fmt.Sprintf("[%s %s]: %s", c.Service, c.Method, c.Message)
 }
 
-// ErrInvalidType is the error returned when the wrong type is given to an
+// Temporary implements net.Error#Temporary
```

Comment:
Do we really need this? we already have public fields (which should be named `Temporary` and `Timeout`)